pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -537,45 +538,45 @@ message ContainerConfig {
     optional bool stdin_once = 13;
     optional bool tty = 14;
 
-    // Linux contains configuration specific to Linux containers.
+    // Configuration specific to Linux containers.
     optional LinuxContainerConfig linux = 15;
 }
 
 message CreateContainerRequest {
-    // The id of the PodSandbox
+    // ID of the PodSandbox in which the container should be created.
     optional string pod_sandbox_id = 1;
-    // The config of the container
+    // Config of the container.
     optional ContainerConfig config = 2;
-    // The config of the PodSandbox. This is the same config that was passed
+    // Config of the PodSandbox. This is the same config that was passed
     // to RunPodSandboxRequest to create the PodSandbox. It is passed again
     // here just for easy reference. The PodSandboxConfig is immutable and
     // remains the same throughout the lifetime of the pod.
     optional PodSandboxConfig sandbox_config = 3;
 }
 
 message CreateContainerResponse {
-    // The id of the created container
+    // ID of the created container.
     optional string container_id = 1;
 }
 
 message StartContainerRequest {
-    // The id of the container
+    // ID of the container to start.
     optional string container_id = 1;
 }
 
 message StartContainerResponse {}
 
 message StopContainerRequest {
-    // The id of the container
+    // ID of the container to stop.
     optional string container_id = 1;
-    // Timeout in seconds to stop the container
+    // Timeout, in seconds, to stop the container.
```

Comment:
it's fine IMO and I expect this will be cleaned up when we address the TODO for what the timeout means